internal/provider/resource_project.go

```diff
@@ -41,14 +49,30 @@ func resourceWizProject() *schema.Resource {
 				Description: "The business unit to which the project belongs.",
 				Optional:    true,
 			},
+			"project_owners": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "A list of project owner IDs.",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"security_champions": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "A list of security champions IDs.",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
 			"slug": {
 				Type:        schema.TypeString,
-				Description: "Short identifier for the project. The value must be unique, even against archived projects, so a uuid is generated and used as the slug value.",
+				Description: "Short identifier for the project. The value must be unique, even against archived projects, so a uuid is generated and used as the slug value. If slug is not provided it will be derived from the name, slug must be unique across all projects (in contrast to name).",
```

Comment:
```slug``` is a computed value and cannot be provided by the user; not following the update to the description.  Did you mean to change ```slug``` to ```optional```?  If you did intend for slug to be optional, the delete operation needs to be updated to alter the slug value to something unique so that project recreation (ForceNew) will be possible.

Reply:
It was from the description in your [branch](https://github.com/AxtonGrams/terraform-provider-wiz/compare/main...project_accounts#diff-46c93ceedaf727b6cb9cca03f61c002560169bc1a49080dfbe3ca297c845af46R67), why I wanted to clarify with you first in the related issue why the uid is being generated.

Reply:
Gotcha.  We can drop that comment.  Time off and I've forgotten where I was on things ;)

Reply:
Sure will do ;)